tests/src/test/java/alluxio/client/fs/ConcurrentFileSystemMasterSetTtlTest.java

```diff
@@ -133,7 +134,9 @@ public void run() {
             AuthenticatedClientUser.set(TEST_USER);
             barrier.await();
             mFileSystem.setAttribute(paths[iteration], SetAttributePOptions.newBuilder()
-                .setTtl(ttls[iteration]).setTtlAction(TtlAction.DELETE).build());
```

Comment:
FileSystem should do merging.

Reply:
We want to specifically set the TTL and Ttl action here which varies from iteration to iteration. Passing options here is required. It's only changed because setAttribute no longer has TTL and TTL action as a top-level attribute. They both exists in common options now